Source/Message.swift

```diff
@@ -94,3 +97,37 @@ extension Message {
         }
     }
 }
+
+extension Message {
+    
+    /// Using `Equatable`'s default implementation is bound to give us false positives since two `Message`s may have semantically equal, but textually different, `jsonData`.
+    ///
```

Comment:
Since comparing messages is an implementation detail, I'd move this to its own extension file, `Message+Equatable`. This will also keep the `Message` interface clean.